README.md

```diff
@@ -529,13 +515,25 @@ We hope to bring these optimizations to GHC in future but until
 then we recommend that you use the plugin for performance sensitive
 applications.
 
-## Support
+## A Note on Internal Modules
 
-Please feel free to ask questions on the
-[streamly gitter channel](https://gitter.im/composewell/streamly).
-If you require professional support, consulting, training or timely
-enhancements to the library please contact
-[support@composewell.com](mailto:support@composewell.com).
+You may see some `Internal` modules imported in the examples. Those are
+not really Internal to the library.  We can classify streamly modules in
+two categories:
+
+* Released Modules: these are modules that have a stable API, any API changes
+  conform to a versioning policy.
+* Pre-release Modules: These modules are not yet released due to some planned
+  changes in near future, they will be released soon. These modules reside in
+  the `Internal` namespace for now.
```

Comment:
I'm unable to phrase this properly. But replace `some planned changes in near future` with something regarding having a soak time for the API.